website/pages/docs/internals/security.mdx

```diff
@@ -0,0 +1,322 @@
+    client’s configuration file can enable and disable features for a client
+    including insecure drivers such as
+    [raw_exec](/docs/drivers/raw_exec.html).
+
+* **Memory access to a running Nomad client agent** - Direct access to the
+    memory of the Nomad client agent process allows an attack to extract secrets
+    from clients such as Vault tokens.
+
+* **Lax Client Driver Sandbox** - Drivers may allow some privileged operations,
+    e.g. filesystem access to configuration directories, or raw accesses to host
+    devices. Such privileges can be used to facilitate compromise other workloads,
+    or cause denial-of-service attacks.
+
+#### Internal Threats
+
+* **Operator** - Someone with a valid mTLS cert and ACL token may still be a
+    threat to your cluster in certain situations, especially in multi-team
+    cluster deployments. They may accidentally or intentionally use a malicious
+    jobspec to harm a cluster which can help be protected against using
+    Namespaces and Sentinel policies.
+
+* **Workload** - Workloads may have host network access within a cluster which
+    can lead to SSRF due to application security issues outside of the scope of
+    Nomad which may lead to internal access within the cluster. Using mTLS, ACLs
+    and Sentinel policies together can add layers of protection against
+    malicious workloads.
+
+* **RPC / API Access** - RPC and HTTP API endpoints without mTLS can expose
+    clusters to abuse within the cluster from malicious workloads.
+
+* **Client driver** - Drivers implement various workload types for a cluster,
+    and the backend configuration of these drivers should be considered to
+    implement defense in depth. For example, a custom Docker driver that limits
+    the ability to mount the host file system may be subverted by network access
+    to an exposed Docker daemon API through other means such as the raw_exec
+    driver.
+
+
+#### External Threats
+
+There are two main components to consider to for external threats in a Nomad cluster:
+
+* **Server agent** - Internal cluster leader elections and replication is
+    managed via Raft between server agents encrypted in transit. However,
+    information about the server is stored unencrypted at rest in the agent’s
+    data directory. This information may contain information such as ACL tokens
+    and TLS certificates.
+
+* **Client agent** - Client-to-server communication within a cluster is
+    encrypted and authenticated using mTLS. Information about the allocations on
+    a client node is unencrypted in the agent’s data and configuration
+    directory.
+
+### Network Ports
+
+
+| **Port / Protocol**  | Agents  | Description |
+|----------------------|---------|-------------|
+| **4646** / TCP       | All     | [HTTP](https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol) to provide [UI](/guides/web-ui/access.html) and [API](/api/index.html) access to agents. |
```

Comment:
I think this is the canonical path? I would imagine we have to maintain the redirects ~forever so it probably doesn't matter.

```suggestion
| **4646** / TCP       | All     | [HTTP](https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol) to provide [UI](/guides/web-ui/access.html) and [API](/api-docs) access to agents. |
```